test/fixtures/files/nextflow/schema_input.json

```diff
@@ -12,6 +12,12 @@
         "pattern": "^\\S+$",
         "meta": ["id"],
         "unique": true,
+        "errorMessage": "Sample id must be provided and cannot contain spaces"
+      },
+      "sample_name": {
+        "type": "string",
+        "pattern": "^\\S+$",
```

Comment:
**nitpick (blocking):** Can you remove this `pattern` for this field, also can you update the description to just `Sample name must be provided`

Reply:
81437489b82227c3c432a318260e7e9219bd83a5